src/IotaPayment.vue

```diff
@@ -41,7 +41,7 @@ export default {
     payWithIota() {
       const self = this;
       axios
-        .post(this.$payOptons.url + this.$payOptons.api_path, this.order)
+        .post(this.$payOptons.url + this.$payOptons.api_path+'/payments', this.order)
```

Comment:
Could the ''/payments' ending not be part of the "api_path" variable?

Reply:
Yes, but if you want to add support for other api calls like /payouts it doesn't make sense there i think